pkg/virtctl/imageupload/imageupload.go

```diff
@@ -139,6 +141,7 @@ func NewImageUploadCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 	cmd.MarkFlagRequired("image-path")
 	cmd.Flags().BoolVar(&noCreate, "no-create", false, "Don't attempt to create a new DataVolume/PVC.")
 	cmd.Flags().UintVar(&uploadPodWaitSecs, "wait-secs", 300, "Seconds to wait for upload pod to start.")
+	cmd.Flags().BoolVar(&forceBind, "force-bind", false, "Force Bind the PVC, ignoring the WaitForFirstConsumer logic")
```

Comment:
`"Bind"` --> `"bind"`?
Also, for consistency with other flags, use a period at the end of the sentence.

Reply:
done